app/models/answer.rb

```diff
@@ -86,4 +86,37 @@ def is_valid?
   def non_archived_notes
     return notes.select{ |n| n.archived.blank? }.sort!{ |x,y| y.updated_at <=> x.updated_at }
   end
+
+
+  ##
+  # Returns the parsed JSON hash for the current answer object
+  # Generates a new hash if none exists for rda_questions
+  #
+  # @return [Hash] the parsed hash of the answer.
+  #                Should have keys 'standards', 'text'
+  #                'standards' is a list of <std_id>: <title> pairs
+  #                'text' is the text from the comments box
+  def answer_hash

```

Comment:
This seems specific to RDA. Would be good to refactor at some point since we may end up with other answer types that need to store as hash objects.

Perhaps extending Answer would be a good approach `class RdaAnswer < Answer` and then override the `text` and `text=` methods